.github/workflows/composites/run-and-save-test-times-when-cache-missing/action.yaml

```diff
@@ -46,7 +46,9 @@ runs:
             -Dhttp.keepAlive=false \
             -Dmaven.wagon.http.pool=false \
             -Dmaven.wagon.http.retryHandler.class=standard \
+            -Dmaven.wagon.httpconnectionManager.ttlSeconds=120 \
```

Comment:
I have observed that we get some timeouts from time to time when we need something from maven central, seems that adding these options fixes the problem.